src/main/java/edu/ucsb/courses/documents/statistics/SingleCourseSearch.java

```diff
@@ -0,0 +1,78 @@
+package edu.ucsb.courses.documents.statistics;
+
+import java.util.List;
+import java.util.Objects;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
These are all unused imports, we should remove them.

Reply:
We removed the unused imports